src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -20,72 +234,105 @@ def test_property_rw():
+    from tests.test_utils.test_utils import generate_add_model
 
-    core.set_property({"ALLOW_AUTO_BATCHING": True})
-    assert core.get_property(properties.hint.allow_auto_batching()) is True
+    model = generate_add_model()
 
+    assert properties.hint.model() == "MODEL_PTR"
 
-@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU",
-                    reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
+    property_tuple = properties.hint.model(model)
+    assert property_tuple[0] == "MODEL_PTR"
+
+
+@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU", reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
 def test_single_property_setting():
     core = Core()
+
+    if "Intel" not in core.get_property("CPU", "FULL_DEVICE_NAME"):
+        pytest.skip("This test runs only on openvino intel cpu plugin")
+
     core.set_property("CPU", properties.streams.num(properties.streams.Num.AUTO))
 
     assert properties.streams.Num.AUTO.to_integer() == -1
     assert type(core.get_property("CPU", properties.streams.num())) == int
 
 
-@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU",
-                    reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
-@pytest.mark.parametrize("properties_to_set", [
-    # Dict from list of tuples
-    dict([  # noqa: C406
-        properties.enable_profiling(True),
-        properties.cache_dir("./"),
-        properties.inference_num_threads(9),
-        properties.affinity(properties.Affinity.NONE),
-        properties.hint.inference_precision(Type.f32),
-        properties.hint.performance_mode(properties.hint.PerformanceMode.LATENCY),
-        properties.hint.num_requests(12),
-        properties.streams.num(5),
-    ]),
-    # Pure dict
-    {
-        properties.enable_profiling(): True,
-        properties.cache_dir(): "./",
-        properties.inference_num_threads(): 9,
-        properties.affinity(): properties.Affinity.NONE,
-        properties.hint.inference_precision(): Type.f32,
-        properties.hint.performance_mode(): properties.hint.PerformanceMode.LATENCY,
-        properties.hint.num_requests(): 12,
-        properties.streams.num(): 5,
-    },
-    # Mixed dict
-    {
-        properties.enable_profiling(): True,
-        "CACHE_DIR": "./",
-        properties.inference_num_threads(): 9,
-        properties.affinity(): "NONE",
-        "INFERENCE_PRECISION_HINT": Type.f32,
-        properties.hint.performance_mode(): properties.hint.PerformanceMode.LATENCY,
-        properties.hint.num_requests(): 12,
-        "NUM_STREAMS": properties.streams.Num(5),
-    },
-])
+@pytest.mark.skipif(os.environ.get("TEST_DEVICE", "CPU") != "CPU", reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
+@pytest.mark.parametrize(
+    "properties_to_set",
+    [
+        # Dict from list of tuples
```

Comment:
I don't think it's a dict from list of tuples, it's a dict from a list.

Reply:
If you use RW property with argument, it will return tuple. So... it is a list of tuples 😃